lib/Renard/Curie/Component/PageDrawingArea.pm

```diff
@@ -288,9 +288,9 @@ method setup_scroll_bindings () {
 	$self->scrolled_window->signal_connect ( scroll_event => \&on_scroll_event_cb, $self );
 }
 
-=func on_scroll_event_cb
+=fun on_scroll_event_cb
```

Comment:
This should stay as `=func`. This is part of the POD directives for the documentation.